<launch>
  <node name="position_moveit" pkg="campusrover_arm_move" type="position_moveit" output="screen" >
		<param name="planning_frame_id" value="link_0"/>
        <param name="planning_group_name" value="arm"/>
        <param name="standby_pose_name" value="standby_pose"/>
        <param name="planning_time" value="5.0"/>
        <param name="num_planning_attempts" value="10"/>
        <param name="shift_x" value="-0.1"/>
        <param name="shift_y" value="-0.08"/>
        <param name="shift_z" value="0.06"/>
        <param name="press_dis" value="0.04"/>
        <param name="jump_threshold" value="0.0"/>
        <param name="eef_step" value="0.002"/>
        <param name="allow_replanning" value="true"/>
        <param name="Visualization" value="true"/>
	</node>
</launch>